### ISSUE TITLE
Tooltips: Consistency and contrast issues

### ISSUE THREAD

**Issue by Tecdoc** [WordPress/gutenberg#11180]
_As a author using the new Editor in WordPress, I expect a consistent experience with tooltips for buttons and elements that use only icons._

@postphotos and I were testing Gutenberg accessibility at WordCamp Philly contributor day and discovered this accessibility issue.

**Description**
Testing the current Gutenberg in the WP 5.0 beta1 version. The Gutenberg element buttons (Top Right of the page editor) should have the tooltip appear below the element button (like the full screen design patter) not above it because when tooltip is above it is difficult to read/differentiate because of the black color admin bar background, and the black background tooltip background.   

**To Reproduce**
1. Hover on each of the four buttons
2. Wait until the tooltip appears 

**Expected behavior**
The goal is to have a consistent design pattern of these button elements' tooltips to appear below the button like the full screen design pattern

**Screenshots**
**Sidebar View**
![gutenberg-tooltips](https://user-images.githubusercontent.com/1907392/47620858-ea5a5080-dac5-11e8-8251-010dc5adf6d4.gif)

**Full Screen View**
![guten-tooltips2](https://user-images.githubusercontent.com/1907392/47620923-b0d61500-dac6-11e8-822e-c9c4662afab3.gif)

**Desktop Tested on the Follow**
 - OS: iOS
 - Browsers: Chrome, Firefox v61 and Sarari

**Smartphone Tested:**
 - Device: Honor View 10
 - OS: Android

**Comment by LukePettway** [comment]
This may also be relevant to #8040

**Comment by karmatosed** [comment]
Where possible they should absolutely be consistent. I think adding this or at least combining inconsistencies into one issue with #8040 makes sense. Let's do that and thank you for reporting.
